django_react_proj/users-fe/src/index.js

```diff
@@ -6,9 +6,9 @@ import reportWebVitals from './reportWebVitals';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <React.Fragment>
```

Comment:
you can use `<></>` everywhere in the app instead of `<React.Fragment></React.Fragment>` (it's the same but easier to read)